src/modules/cpu/kernel/crop_mirror_normalize.hpp

```diff
@@ -23,15 +23,12 @@ RppStatus crop_mirror_normalize_u8_u8_host_tensor(Rpp8u *srcPtr,
         RpptROIPtr roiPtrInput = &roiTensorPtrSrc[batchCount];
         compute_roi_validation_host(roiPtrInput, &roi, &roiDefault, roiType);
 
-        std::vector<float> mean(srcDescPtr->c), invStdDev(srcDescPtr->c);
         Rpp32u incrementPerImage = srcDescPtr->c * batchCount;
         __m256 pCMNParams[2 * srcDescPtr->c];
```

Comment:
This could also be:
```
Rpp32u cmnParamLoc = srcDescPtr->c * batchCount;
Rpp32s numRegs = 2 * srcDescPtr->c;
__m256 pCMNParams[numRegs];
for(int pos = 0; pos < numRegs; pos += 2)
{
    pCMNParams[pos] = _mm256_set1_ps(multiplierTensor[cmnParamLoc]);
    pCMNParams[pos + 1] = _mm256_set1_ps(offsetTensor[cmnParamLoc]);
    cmnParamLoc++;
}
```